.azure/pipelines/helix-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     agentOs: Windows
     timeoutInMinutes: 240
     steps:
-    - script: .\build.cmd -all -ci /t:Helix /p:IsHelixJob=true /bl:$env:BUILD_SOURCESDIRECTORY\artifacts\log\SendToHelix.binlog
+    - script: .\build.cmd -all -ci /p:BuildNative=true /t:Helix /p:IsHelixJob=true /bl:$env:BUILD_SOURCESDIRECTORY\artifacts\log\SendToHelix.binlog
```

Comment:
@natemcmaster this was the work around I decided on for now. Modifying the scripts required modifying a few scripts and had oddities. As we are removing the eng folder eventually, we can remove this in the future as repo.props will be deleted.

Reply:
thanks for the heads up